### PR TITLE
Initialize repository

### PR DESCRIPTION
**Pull request by prog893**
Initial implementation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,120 @@
...
+}
+
+resource "aws_kinesis_stream" "stream" {
+  name             = "stream"
+  shard_count      = 1
+  retention_period = 24
+}
+
+module "fluent" {
+  source  = "baikonur-oss/lambda-kinesis-to-s3/aws"
```
Comment (toVersus) [inline README.md]:
```suggestion
  source  = "baikonur-oss/lambda-kinesis-to-fluent/aws"
```

File: lambda/main.py [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+FAILED_LOG_S3_BUCKET = os.environ['FAILED_LOG_S3_BUCKET']
+FAILED_LOG_S3_PATH_PREFIX = os.environ['FAILED_LOG_S3_PREFIX']
+
+LOG_TYPE_FIELD: str = os.environ['LOG_TYPE_FIELD']
+LOG_TIMESTAMP_FIELD: str = os.environ['LOG_TIMESTAMP_FIELD']
+LOG_TYPE_FIELD_WHITELIST: set = set(str(os.environ['LOG_TYPE_WHITELIST']).split(','))
+LOG_TYPE_UNKNOWN_PREFIX: str = os.environ['LOG_TYPE_UNKNOWN_PREFIX']
+
+LOG_FLUENT_TARGET_URL = os.environ['LOG_FLUENT_TARGET_URL']
+LOG_FLUENT_TAG = os.environ['LOG_FLUENT_TARGET_URL']  # second-level tag
```
Comment (toVersus) [inline lambda/main.py]:
```suggestion
LOG_FLUENT_TAG = os.environ['LOG_FLUENT_TAG']  # second-level tag
```

File: README.md [hunk 3]
```diff
@@ -1,32 +1,120 @@
...
+  version = "1.0.0"
+
+  lambda_package_url = "https://github.com/baikonur-oss/terraform-aws-lambda-kinesis-fluent/releases/download/v1.0.0/lambda_package.zip"
+  name               = "kinesis_to_fluent"
+
+  batch_size         = 100
+  kinesis_stream_arn = "${aws_kinesis_stream.stream.arn}"
+  fluent_tag         = "dev"
+  fluent_target_url  = "10.20.30.40"
```
Comment (toVersus) [inline README.md:62]: IMO, `fluent_target_hostname` is more suitable for representing the expected value from the user.

File: lambda/main.py [hunk 4]
```diff
@@ -0,0 +1,284 @@
...
+# logger = logging.getLogger()
+# logger.setLevel(logging.DEBUG)
+
+# patch boto3 with X-Ray
+libraries = ('boto3', 'botocore')
+patch(libraries)
+
+# global client instances
+s3 = boto3.client('s3')
+os.environ['TZ'] = "Asia/Tokyo"
```
Comment (toVersus) [inline lambda/main.py]: Remove this line.

File: README.md (hunk 3 above)
Comment (prog893) [reply README.md:62]: Maybe `fluent_sink_endpoint` would be better? This could be a hostname or could be an IP address.

**Comment by prog893** [comment]
Addressed PR comments

File: README.md (hunk 3 above)
Comment (toVersus) [reply README.md:62]: I see, it sounds reasonable.

**Comment by prog893** [comment]
:eyes:

**Review by toVersus — APPROVED** [review]
LGTM (Sorry for my delayed response.)

**Pull request merged**